analyzers/its/regression-test.ps1

```diff
@@ -524,23 +506,6 @@ catch {
 finally {
     Pop-Location
 
-    foreach ($msBuildVersion in $msBuildVersions) {
```

Comment:
review note: this is not necessary anymore because the Directory.Build.targets is in the folder of each project, and the workspace gets cleaned after each CI execution

https://github.com/SonarSource/sonar-dotnet/blob/2d27f118394b3b7d31d9f4a21c9953d777110d52/azure-pipelines.yml#L63-L64